src/views/Profile/AdminTable/AdminFilter.tsx

```diff
@@ -51,29 +51,18 @@ export const AdminFilter: React.FC<IMaterialsFilter> = ({
   const classes = useStyles();
   const { t } = useTranslation();
   const [open, setOpen] = useState(false);
-  const selectRef = useRef(null);
 
   useEffect(() => {
-    const handleScroll = () => {
-      if (open) {
-        setOpen(false);
-      }
-    };
+    const handleScroll = () => handleClose();
```

Comment:
you could directly use `handleClose` but using `handleScroll` as a wrapper around it is also fine

in my opinion